### DISCUSSION TITLE
Implement the timestamp datatype for MySQL/MariaDB

### DISCUSSION THREAD

**Discussion by jonas-dongowski** [vlang/v#18682] [Ideas]
### Describe the feature

Currently when trying to retrieve data from a table that contains a TIMESTAMP field, the following error is shown:
`V panic: 'timestamp' is not yet implemented. Please create a new issue at https://github.com/vlang/v/issues/new`

### Use Case

There is software out there that is using the timestamp data type. To successfully integrate with their databases using V, this implementation is necessary.

### Proposed Solution

Implement the timestamp datatype. Ideally with the ability to deal with such a value as a time.Time variable.

### Other Information

I've tested using a MariaDB instance. It's possible that this fact lead to the issue as I've not tested using a MySQL instance.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### Version used

https://github.com/jonas-dongowski/v/commit/1359e9da408d80fca4a91370ba7a1e9f18d9cf15

### Environment details (OS name and version, etc.)

Docker version 20.10.13, build a224086

V full version: V 0.3.4 1be798b.3a09142
OS: linux, Linux version 5.10.16.3-microsoft-standard-WSL2 (oe-user@oe-host) (x86_64-msft-linux-gcc (GCC) 9.3.0, GNU ld (GNU Binutils) 2.34.0.20200220) #1 SMP Fri Apr 2 22:23:49 UTC 2021 (WSL 2)
Processor: 16 cpus, 64bit, little endian, AMD Ryzen 7 2700X Eight-Core Processor

getwd: /app
vexe: /opt/vlang/v
vexe mtime: 2023-05-22 07:28:00

vroot: OK, value: /opt/vlang
VMODULES: OK, value: /root/.vmodules
VTMP: OK, value: /tmp/v_0

Git version: git version 2.20.1
Git vroot status: weekly.2023.19-48-g3a09142a
.git/config present: true

CC version: cc (Debian 8.3.0-6) 8.3.0
thirdparty/tcc status: thirdparty-linux-amd64 12f392c3

**Comment by ghost** [comment]
Fixed in #18704
